wrappers/src/query_cs.cpp

```diff
@@ -391,4 +391,33 @@ REALM_EXPORT Results* query_create_sorted_results(Query * query_ptr, SharedRealm
     });
 }
 
+
+#pragma mark  ObjectId Searches
+// These use the faster calls such as find_first_int rather than having to build and use a query in one go.
+// Note to bypass the overhead of looking up the PrimaryKey field by name,
```

Comment:
This seems like a weird place to put this comment. The behavior it is describing lives in the callers of these functions.

I also question the value of having a comment describing Cocoa's current behavior since it's very likely to get out of date, and unlikely to be updated when that happens.